lib/still/preprocessor.ex

```diff
@@ -148,13 +155,18 @@ defmodule Still.Preprocessor do
   end
 
   defp preprocessors do
-    Map.merge(@default_preprocessors, user_defined_preprocessors())
+    preprocessors_to_list(user_defined_preprocessors()) ++
+      preprocessors_to_list(@default_preprocessors)
   end
 
   defp user_defined_preprocessors do
     config(:preprocessors, %{})
   end
 
+  defp preprocessors_to_list(preprocessors) do
+    Enum.map(preprocessors, fn {key, value} -> {key, value} end)
```

Comment:
I'm confused. How this different from `Enum.to_list/1`?

Reply:
It's not, my previous implementation was different and I changed the code without realizing that 👍